.github/workflows/phpunit.yml

```
@@ -9,11 +9,17 @@ jobs:
       # do not stop on another job's failure
       fail-fast: false
       matrix:
-        php-versions: ['8.0', '8.1', '8.2']
+        php-versions: ['8.2', '8.3']
         coverage: ['xdebug']
         xdebug_mode: ['debug']
     name: php${{ matrix.php-versions }}
     steps:
+      - name: Install Dependencies
+        run: |
+          apt-get update
+          apt-get install -y espeak-ng
```

Reference: https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners/customizing-github-hosted-runners#installing-software-on-ubuntu-runners

```suggestion
          sudo apt-get update
          sudo apt-get install -y espeak-ng
```